integrations/java/workload-executor

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+
+export JAVA_HOME="/opt/java/jdk11"
```

Comment:
I think you should define this in the `runtime` axis to make it easy to test using other JDK versions going-forward. Both node and Python already do this.

Reply:
I don't see a need to test any other JDKs.  Can we leave it here until we have the need?

Reply:
If I did do it that way, would the variable I define there be accessible in this script?  e.g. 

    export JAVA_HOME="/opt/java/$JDK. 

where the runtime access had

   JDK: "jdk11"

?

Reply:
Yes. You could actually just define `JAVA_HOME` in its entirety in the `runtime` axis so this `export` statement wouldn't be needed at all.

Reply:
Ok, you convinced me.  Done.